environment-dev.yml

```diff
@@ -37,7 +37,7 @@ dependencies:
   # pymssql==2.2.8 broken: https://github.com/modin-project/modin/issues/6429
   - pymssql>=2.1.5,!=2.2.8
   - psycopg2>=2.8.6
-  - fastparquet>=0.6.3,<2023.1.0
+  - fastparquet>=0.6.3
```

Comment:
I wonder if we should really bump the _lower_ bound instead, to something like `>2023.1.0`?.. as, if I understand correctly, earlier versions of `fastparquet` aren't properly compatible with pandas 2.0.

Reply:
On the one hand, yes, on the other hand, pandas still supports fairly old versions. I guess there are rare compatibility issues, so they don't raise the minimum version. I think we should do the same.

Reply:
Yeah, we should probably just mirror what pandas requires on `fastparquet`...